docs/tiered-storage.md

```
@@ -0,0 +1,346 @@
+sequentially within a 55-minute wall-clock budget — no per-blob
+fan-out. Each row is offloaded under the per-sha advisory lock, so
+the row flip is atomic. Whatever isn't done in one tick is picked up
+by the next.
+
+## Encryption
+
+To enable encryption-at-rest for blobs:
+
+1. Generate a random secret string. Use `openssl rand` or equivalent
+   so the input is genuinely high-entropy:
+   ```sh
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)